molns.py

```diff
@@ -1454,6 +1454,29 @@ def run(self, args, config_dir=None):
             Command('import',{'filename.json':None},
                 function=MOLNSProvider.provider_import),
         ]),
+                
+        SubCommand('service',[
```

Comment:
We should discuss this.  This would make 'service' in the molns vocabulary as a clone/alias of 'provider'.  What is the motivation to do that?